### DISCUSSION TITLE
Exiting notifications screen - create easier way

### DISCUSSION THREAD

**Discussion by Zelectrician** [home-assistant/frontend#21238] [Other feature requests]
When you click notifications a box pops up and you must click the X at the very top right of the screen. If you have ghost notifications or accidentally click on notifications it would be nice if you could re-click the notifications bell in order to re-exit the menu instead of having to navigate to the top of the screen. Running my monitor in portrait mode so cursor has a ways to go to exit.

Not a huge deal but would be more intuitive and easier to collapse notifications. Currently i believe the notifications window blocks the notifications bell. Not sure if other people find this annoying, but if it's not to hard to implement, it would make a great UI improvement.

**Comment by karwosts** [comment]
You can also just click anywhere outside the notification bar to dismiss it?
